mern-ecommerce/Dockerfile

```diff
@@ -3,4 +3,4 @@ WORKDIR /app
 COPY ./package.json .
 COPY . .
 RUN npm install
-CMD ["node", "server.js"]
\ No newline at end of file
+CMD ["node", "servers.js"]
```

Comment:
The command `CMD ["node", "servers.js"]` should be in a separate line and not at the end of the Dockerfile. It should be the last command in the Dockerfile as it is the default command to run when the container starts. 

Additionally, it would be better to specify the working directory before running the command, to ensure that the command is executed in the correct directory. 

The corrected code should be:
```
WORKDIR /app
CMD ["node", "servers.js"]
```